migration-tests/src/lib.rs

```diff
@@ -0,0 +1,74 @@
+use futures::executor::block_on;
+use polymesh_runtime::{DryRunRuntimeUpgrade, Runtime};
+use remote_externalities::{Builder, CacheConfig, Mode, OfflineConfig, OnlineConfig};
+use remote_externalities_old::{
+    Builder as BuilderOld, Mode as ModeOld, OfflineConfig as OfflineConfigOld,
+};
+use sp_core::storage::{StorageData, StorageKey};
+use sp_state_machine_old::backend::Backend;
+use std::{sync::Mutex, time::Instant};
+
+#[macro_use]
+extern crate lazy_static;
+
+#[cfg(test)]
+mod two_five_zero;
+
+struct TestState;
+
+impl TestState {
+    fn init() {
+        if !std::path::Path::new(".").join("CACHE").exists() {
+            block_on(
+                Builder::new()
+                    .mode(Mode::Online(OnlineConfig {
+                        uri: "http://159.69.94.51:9933".into(),
```

Comment:
Do we want this URL to open? isn't it better to have some environment variables?

Reply:
A chance of DDoS

Reply:
Think of it as a honey pot. I'd rather have the dumb script kiddies dos this than our other infra :)